GetDKAN/dkan
# Change the default title for resources to use the format rather than url

Connects #2027

Harvested resources that do not have title values are using the url as the title. Switching to use the format value instead.

## User story

When I harvest data from a socrata source, I don't want to see long urls used as the resource titles, very confusing.

## How to reproduce

1. Create a harvest source `node/add/harvest-source` with https://data.medicare.gov/data.json
2. Filter by `keyword` = `doctor`
3. Harvest, notice that the resources have titles like **https://data.medicare.gov/api/views/hanv-ru8h/rows.csv?accessType=DOWNLOAD**

## QA Steps

1. Create a harvest source `node/add/harvest-source` with https://data.medicare.gov/data.json
2. Filter by `keyword` = `doctor`
3. Harvest, notice that the resources have titles that use the format value as the title

## Deployment Steps

Existing harvests will only update if the hash changes, you can force an update with `drush dkan-hm [source_machine_name] --skiphash`

## Reminders
